NitroxClient/Communication/Packets/Processors/VehicleNameChangeProcessor.cs

```diff
@@ -18,12 +19,36 @@ public VehicleNameChangeProcessor(IPacketSender packetSender)
 
         public override void Process(VehicleNameChange namePacket)
         {
-            GameObject target = NitroxEntity.RequireObjectFrom(namePacket.Id);
-            SubNameInput subNameInput = target.RequireComponentInChildren<SubNameInput>();
+            GameObject vehicleObject = NitroxEntity.RequireObjectFrom(namePacket.VehicleId);
+            SubNameInput subNameInput = null;
+            SubName subNameTarget = null;
 
-            using (packetSender.Suppress<VehicleNameChange>())
+            if (vehicleObject.GetComponent<Vehicle>())
```

Comment:
Can't you use `if (namePacket.ParentId.HasValue)` here as it would fail anyway if it's not a vehicle later?